lib/main_common.pm

```diff
@@ -534,8 +545,13 @@ sub load_system_role_tests {
         loadtest "installation/logpackages";
     }
     loadtest "installation/disable_online_repos" if get_var('DISABLE_ONLINE_REPOS') && !get_var('OFFLINE_SUT');
-    loadtest "installation/installer_desktopselection" if is_opensuse;
-    loadtest "installation/system_role" if is_caasp('kubic');
+    if (is_leap('<=15.0')) {
```

Comment:
So for Leap 15.1 nothing would be triggered now?

Reply:
I guess we can adjust this condition `elsif (is_tumbleweed || is_caasp('kubic'))` according new availability of the changes.

Reply:
good point!

Reply:
Right, that was not addressed neither here nor in #5547. I would consider reverting, as PR got merged too fast

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5551